source/content/modules-known-issues.md

```diff
@@ -395,10 +395,14 @@ ___
 ___
 
 ## [Simple OAuth / OAuth 2.0](https://www.drupal.org/project/simple_oauth)
+<ReviewDate date="2024-06-12" />
 
-**Issue**: The module requires a very specific set of permissions for the folder and the keys to be uploaded. Using Private or non-standard filepaths won't work. It is not possible to change these in LIVE or TEST environment.
+**Issue**: This module requires a very specific set of permissions for the folder and the keys to be uploaded.
+
+**Solution**: Apply the following patches (which have been tested and validated by Pantheon), based on your site's PHP version, to solve for this issue:
+* [Oauth Permission Patch - Required to resolve Pantheon known issues (PHP 8.0+)](https://patch-diff.githubusercontent.com/raw/lcatlett/oauth2-server/pull/1.patch)
+* [Oauth Permission Patch - Required to resolve Pantheon known issues (PHP 7.4)](https://patch-diff.githubusercontent.com/raw/lcatlett/oauth2-server/pull/2.patch)
```

Comment:
```suggestion
* [Oauth Permission Patch - Required to resolve Pantheon known issues (PHP 7.4)](https://patch-diff.githubusercontent.com/raw/pantheon-systems/oauth2-server/pull/2.patch)
```

Reply:
@stevector I think you might have duplicated the first PR twice by mistake? 

https://patch-diff.githubusercontent.com/raw/lcatlett/oauth2-server/pull/2.patch

vs

https://patch-diff.githubusercontent.com/raw/lcatlett/oauth2-server/pull/2.patch

Reply:
Not exactly, I think @lcatlett's diff is bigger because it involves a merge commit too?

We probably need @lcatlett's expertise to explain why these patches are necessary. I don't understand why PHP different PHP versions need different patches.

Reply:
Ah gotcha, my mistake I'll go ahead and commit this suggestion which I think will resolve the convo. I'll reassign to you for next steps (defer to you on whether we merge or proceed with additional tech review first)